site/content/en/docs/tasks/manage/setup_multikueue.md

```diff
@@ -100,3 +100,9 @@ CQ - Active: True Reason: Ready Message: Can admit new workloads
 AC - Active: True Reason: Active Message: The admission check is active
 MC - Active: True Reason: Active Message: Connected
 ```
+
+## Setup MultiKueue with Open Cluster Management
+
+[Open Cluster Management](https://open-cluster-management.io/) (OCM) is a community-driven project focused on multicluster and multicloud scenarios for Kubernetes apps. Open APIs are evolving within this project for cluster registration, work distribution, dynamic placement of policies and workloads, and much more.
```

Comment:
I'm not sure what "Open APIs" means.

Reply:
Updated the description for ocm and removed "Open APIs" to avoid confusion.